### PR TITLE
Update ray-operator documentation and image version in ray-cluster.heterogeneous.yaml

### PR DESCRIPTION
**Pull request by jasoonn**
## Why are these changes needed?
The example program doesn't get executed as specified in the readme. This PR updated the readme to give users instructions to execute the example program and show what the result should look like. Besides, this PR upgrades the image to 2.0.0 to allow submitting jobs to Ray cluster, and removes object-manager-port and node-manager-port configuration to avoid port conflicts.

## Related issue number

Closes https://github.com/ray-project/kuberay/issues/583
## Checks

- [ ] I've made sure the tests are passing. 
- Testing Strategy
   - [ ] Unit tests
   - [x] Manual tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: ray-operator/config/samples/ray-cluster.heterogeneous.yaml [hunk 1]
```diff
@@ -24,8 +24,8 @@ spec:
     # the following params are used to complete the ray start: ray start --head --block --redis-port=6379 ...
     rayStartParams:
       port: '6379'
-      object-manager-port: '12345'
```
Comment (kevin85421) [inline ray-operator/config/samples/ray-cluster.heterogeneous.yaml:27]: Can you explain the reason of removing these two lines?

Comment (DmitriGekhtman) [reply ray-operator/config/samples/ray-cluster.heterogeneous.yaml:27]: These args are certainly optional, unnecessary, and distracting -- I definitely think it's a good idea to remove them, along with the commented out versions on lines 27 and 28.

Comment (jasoonn) [reply ray-operator/config/samples/ray-cluster.heterogeneous.yaml:27]: @kevin85421 @DmitriGekhtman  Thanks for the comment. I have deleted these two lines.

**Review by kevin85421 — APPROVED** [review]
LGTM

**Review by DmitriGekhtman — APPROVED** [review]
Thanks!

**Pull request merged**